src/QsCompiler/LanguageServer/Program.cs

```diff
@@ -136,9 +145,11 @@ private static int Run(Options options)
             QsLanguageServer server;
             try
             {
-                server = options.ReaderPipeName != null && options.WriterPipeName != null
-                    ? ConnectViaNamedPipe(options.WriterPipeName, options.ReaderPipeName, options.LogFile)
-                    : ConnectViaSocket(port: options.Port, logFile: options.LogFile);
+                server = options.UseStdInOut
```

Comment:
This order of precedence makes sense, standard input -> named pipe -> socket, but since we have independent options set for each, I wonder if we should return an error instead if a user specifies more than one option at the same time.

Reply:
That could make sense, yeah. My understanding was that that's handled by the option set functionality itself, but I could be wrong there.